pyairtable/orm/model.py

```diff
@@ -209,7 +209,8 @@ def save(self) -> bool:
         """
         table = self.get_table()
         record = self.to_record()
-        fields = record["fields"]
+        _fndm = self._field_name_descriptor_map()
```

Comment:
I personally avoid such abbreviations in variables. Of course, you can the the name on the right hand side, but it leads to abuse in further commits. Also I'd like to have a comment on what the loop does. Maybe it makes sense to split this one liner into multiple lines but explain it better.

Reply:
Sure. How about this?

```suggestion
        # Filter fields by read_only attribute so we only set values for writable fields
        map_ = self._field_name_descriptor_map()
```